save-common/src/commonMain/kotlin/org/cqfn/save/core/plugin/Plugin.kt

```diff
@@ -158,6 +159,28 @@ abstract class Plugin(
         }
     }
 
+    /**
+     * Substitute placeholders in `this.execFlags` with values from provided arguments
+     *
+     * @param execFlags a command that will be executed to check resources and emit warnings
+     * @param extraFlags [ExtraFlags] to be inserted into `execFlags`
+     * @param fileNames file name or names, that need to be inserted into `execFlags`
+     * @return `this.execFlags` with resolved placeholders
+     */
+    fun resolvePlaceholdersFrom(
```

Comment:
I think this method doesn't use any data from Plugin class. We could make it a top-level function somewhere near ExtraFlagsExtractor or ExtraFlags class declaration, this will also making testing it easier (it won't require creation of Plugin instance).

Reply:
Yes. Fixed